Sasquatch/SasquatchObjC/AppCenterDelegateObjC.h

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import <UIKit/UIKit.h>
```

Comment:
Why do you need these?

Reply:
to compile without modules enabled. Swift bindings contains some symbols from this